catroid/src/main/java/org/catrobat/catroid/common/Constants.java

```diff
@@ -39,7 +39,7 @@
 
 public final class Constants {
 
-	public static final double CURRENT_CATROBAT_LANGUAGE_VERSION = 1.0;
+	public static final double CURRENT_CATROBAT_LANGUAGE_VERSION = 1.01;
```

Comment:
```suggestion
	public static final double CURRENT_CATROBAT_LANGUAGE_VERSION = 1.02;
```
Because of merge of https://jira.catrob.at/browse/CATROID-849 since last commit.